TestCoreApp_Elliott/Program.cs

```diff
@@ -26,9 +26,10 @@
 
 app.UseAuthorization();
 
-app.MapControllerRoute(
+app.MapAreaControllerRoute(
 	name: "olympics",
-	pattern: "Olympics/{area=Olympics}/{controller=Olympics}/{action=Index}/cat/{activeCat}/game/{activeGame}");
+    areaName: "olympics",
+	pattern: "{controller=Olympics}/{action=Index}/cat/{activeCat}/game/{activeGame}");
```

Comment:
Here is how you add the area to your routing. Notice that I removed the start of the pattern since it automatically adds the area name.